GetService/handler.ts

```diff
@@ -137,13 +122,24 @@ export function GetServiceHandler(
   apiClient: ReturnType<APIClient>
 ): IGetServiceHandler {
   return (_, apiAuth, ___, ____, serviceId) => {
-    return getServiceTask(apiClient, serviceId, apiAuth.subscriptionId)
-      .chain(service =>
-        getSubscriptionKeysTask(apiClient, serviceId).map(subscriptionKeys =>
-          ResponseSuccessJson({
-            ...service,
-            ...subscriptionKeys
-          })
+    return fromEither<ErrorResponses, {}>(
```

Comment:
I think you can use `fromPredicate`

Reply:
Fixed in [19e20f4](https://github.com/pagopa/io-functions-services/pull/68/commits/19e20f4ea840ea6a6fbbcdf1a7a3742b756957aa)